.github/workflows/release.yml

```diff
@@ -28,7 +28,7 @@ jobs:
         run: curl -sSL https://install.python-poetry.org | python3 -
 
       - name: Check pyproject.toml & poetry.lock are in sync
```

Comment:
this is redundant now that we check it in pre-commit but going to leave it here anyway to be extra safe